### ISSUE TITLE
Terrain cracking

### ISSUE THREAD

**Issue by mramato** [AnalyticalGraphicsInc/cesium#3710]
Noticed this while testing #3706

![image](https://cloud.githubusercontent.com/assets/1495196/13789400/3013ab0c-eaba-11e5-962f-833349154be2.png)

Here's the code:

``` javascript
    var viewer = new Cesium.Viewer('cesiumContainer', {
        baseLayerPicker: false,
        terrainProvider: new Cesium.CesiumTerrainProvider({
            url: '//assets.agi.com/stk-terrain/world'
        })
    });

    var rect = Cesium.Rectangle.fromDegrees(-85.0, 35.99, -84.99, 36.0);
    viewer.camera.setView({destination: rect});

    Sandcastle.addToolbarButton('Bug', function () {
        viewer.scene.groundPrimitives.add(new Cesium.GroundPrimitive({
            geometryInstances : new Cesium.GeometryInstance({
                geometry : new Cesium.RectangleGeometry({
                    rectangle : rect
                }),
                attributes: {
                    color: Cesium.ColorGeometryInstanceAttribute.fromColor(new Cesium.Color(1.0, 1.0, 0.0, 0.5))
                }
            }),
            asynchronous : false
        }));
    });
```

**Comment by mramato** [comment]
Entering `110 45` in the Geocoder also reproduces cracking, as reported in #4698

**Comment by hpinkos** [comment]
Also reported here: https://groups.google.com/d/msg/cesium-dev/Iucanef6sBQ/KF3TE91wBAAJ

**Comment by Geoneer** [comment]
I thought this might occus in the specific browser because of the handling of the Globe in webgl. But this is not the case. I did test this in all possible browsers: firefox / IE / Chrome.

Strange thing is at a specific project this is not happening. But I can't get on this project now. (it's on a cruiseship out of internet region). So hopefully I can access this server tomorrow so I can see why this does not happen on that specific server.

Thanks!

**Comment by duvifn** [comment]
see [#3411 (comment)](https://github.com/AnalyticalGraphicsInc/cesium/issues/3411#issuecomment-271267461)

**Comment by hpinkos** [comment]
Reported here: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/lRQ1vBmrSnM

**Comment by mramato** [comment]
I can confirm that #5021 fixes the original cracking problem, but entering `110 45` in the geocoder exposes some major culling issues with terrain, (which may possibly be fixed by #4969)

**Comment by bagnell** [comment]
The terrain cracking is fixed. Going to `110 45` with the geocoder loads fine when using the `EllipsoidTerrainProvider` for me. If I switch it to use STK Terrain, some of the tiles are missing.
